.github/workflows/lint-chart.yaml

```diff
@@ -0,0 +1,61 @@
+# validate any chart changes under charts directory
+name: Chart Lint
+
+env:
+  HELM_VERSION: v3.8.1
+  KIND_VERSION: v0.12.0
+  K8S_VERSION: v1.23.4
+
+on:
+  push:
+  pull_request:
+    paths:
+      - "charts/karmada/**"
+
+jobs:
+  chart-lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v2.1
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      # Python is required because `ct lint` runs Yamale (https://github.com/23andMe/Yamale) and
+      # yamllint (https://github.com/adrienverge/yamllint) which require Python
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.2.1
+        with:
+          version: v3.6.0
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$( ct list-changed )
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --debug --check-version-increment=false
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.2.0
+        if: steps.list-changed.outputs.changed == 'true'
+        with:
+          wait: 120s
+          kubectl_version: ${{ env.K8S_VERSION }}
```

Comment:
I suppose we can add `version` and `node_image` here.

Reply:
https://github.com/helm/kind-action#inputs

Reply:
👌, done